#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile.dpatch by  <giskard@debian.org
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add Makefile for install task

@DPATCH@
diff -urNad mcollective-0.4.1~/Makefile mcollective-0.4.1/Makefile
--- mcollective-0.4.1~/Makefile	1970-01-01 00:00:00.000000000 +0000
+++ mcollective-0.4.1/Makefile	2010-01-06 13:43:56.000000000 +0000
@@ -0,0 +1,38 @@
+#!/usr/bin/make -f
+
+DESTDIR=
+
+clean:
+
+install: install-bin install-lib install-conf install-plugins install-doc
+	
+install-bin:
+	install -d $(DESTDIR)/usr/sbin
+	cp mc-* $(DESTDIR)/usr/sbin
+	cp mcollectived.rb $(DESTDIR)/usr/sbin/mcollectived
+
+install-lib:
+	install -d $(DESTDIR)/usr/lib/ruby/vendor_ruby/
+	cp -a lib/* $(DESTDIR)/usr/lib/ruby/vendor_ruby/
+
+install-conf:
+	install -d $(DESTDIR)/etc/mcollective/
+	install -d $(DESTDIR)/etc/init.d
+	cp etc/* $(DESTDIR)/etc/mcollective/
+	cp mcollective.init $(DESTDIR)/etc/init.d/mcollective
+
+install-plugins:
+	install -d $(DESTDIR)/usr/share/mcollective/
+	cp -a plugins $(DESTDIR)/usr/share/mcollective/
+
+install-doc:
+	install -d $(DESTDIR)/usr/share/doc/
+	cp -a doc  $(DESTDIR)/usr/share/doc/mcollective
+
+uninstall:
+	rm -f $(DESTDIR)/usr/sbin/mcollectived
+	rm -rf $(DESTDIR)/usr/lib/ruby/vendor_ruby/mcollective*
+	rm -rf $(DESTDIR)/usr/share/mcollective
+	rm -rf $(DESTDIR)/etc/mcollective
+
+.PHONY: clean install uninstall
